### PR TITLE
refactor(sensing-imu-corrector): rework parameters

### PR DESCRIPTION
**Pull request by karishma1911**
## Description

Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the sensing_imu_corrector

1. create schema

## Tests performed

No Tests performed

## Effects on system behavior

More reliable and faster parameter configuration file creation.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: sensing/imu_corrector/schema/imu_corrector.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    "velodyne_monitor": {
+      "type": "object",
+      "properties": {
+        "angular_velocity_offset_x": {
+          "type": "number",
+          "description": "roll rate offset in imu_link [rad/s]."
+        },
+        "angular_velocity_offset_y": {
+          "type": "number",
+          "description": "itch rate offset imu_link [rad/s]."
```
Comment (kminoda) [inline sensing/imu_corrector/schema/imu_corrector.json:15]: typo
```suggestion
          "description": "pitch rate offset imu_link [rad/s]."
```

**Review by kminoda — COMMENTED** [review]
Thank you for the contribution!

File: sensing/imu_corrector/schema/imu_corrector.json (hunk 1 above)
Comment (karishma1911) [reply sensing/imu_corrector/schema/imu_corrector.json]: Thank you for suggestion..I updated the description  properly..

**Comment by kminoda** [comment]
Would you fill the following parts in description other than "not applicable."?
- Tests performed
- Effects on system behavior

**Comment by karishma1911** [comment]
> Would you fill the following parts in description other than "not applicable."?
> 
> * Tests performed
> * Effects on system behavior

okay,sure.

**Review by kminoda — APPROVED** [review]

**Pull request merged**